src/content/overview/architecture/authentication/index.md

```diff
@@ -27,6 +27,10 @@ Dex acts as a portal to other identity providers (idP) through connectors. The s
 
 We utilize Kubernetes-native RBAC to control access to resources in the platform API. Resources are segregated into organizations, each represented by a dedicated namespace, enabling improved access control. This approach allows for fine-grained permissions at both the organization and namespace levels. For more detailed information on this topic, you can refer to our comprehensive [multi-tenancy documentation]({{< relref "/overview/fleet-management/multi-tenancy" >}}).
 
+### Observability
+
+The observability platform API operating on the management cluster provides you with read access (through the management cluster Grafana) to the observability data collected in managed clusters as well as any additional observability data you choose to ingest in the observability platform. For more detailed information on this topic, you can refer to our comprehensive [grafana organization documentation]({{< relref "/tutorials/observability/multi-tenancy/creating-grafana-organization" >}}).
```

Comment:
Hmm it's not really an API yet with which customers can interact, right? I mean yeah we're working on that but at this point I think this part is a bit confusing. I would push more for something like "you can access our grafana with our sso" - but I'm not sure this adds any value here, wdyt?

Reply:
The GrafanaOrganizations CRs are an API though right?

Reply:
Well...technically yes... :D If you think this is necessary or adds value then let's keep it in.